pkg/utils/gardener/backupentry.go

```diff
@@ -23,8 +24,11 @@ import (
 var backupEntryDelimiter = "--"
 
 // GenerateBackupEntryName returns BackupEntry resource name created from provided <seedNamespace> and <shootUID>.
-func GenerateBackupEntryName(shootTechnicalID string, shootUID types.UID) string {
-	return shootTechnicalID + backupEntryDelimiter + string(shootUID)
+func GenerateBackupEntryName(shootTechnicalID string, shootUID types.UID) (string, error) {
+	if shootTechnicalID == "" || shootUID == "" {
+		return "", fmt.Errorf("can't generate backup entry name with an empty shoot technical ID or UID")
```

Comment:
Can you split this check and error message to make it clear in the logs which field was actually empty?

Reply:
As you wish.